src/components/ImportJobs/components/FileUploader/FileUploader.js

```diff
@@ -11,43 +11,35 @@ import css from './FileUploader.css';
 
 const cx = classNames.bind(css);
 
-const getTitleClassName = dropZoneState => {
-  return cx({
+const FileUploader = props => {
+  const {
+    title,
+    uploadBtnText,
+    accept,
+    isDropZoneActive,
+    className,
+    acceptClassName,
+    activeClassName,
+    rejectClassName,
+    disabledClassName,
+    maxSize,
+    children,
+    style,
+    getDataTransferItems,
+    onDrop,
+    onDragEnter,
+    onDragLeave,
+  } = props;
+  const titleClassName = cx({
```

Comment:
Please rename

Reply:
Could you please clarify what should be renamed and why?